src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -122,12 +122,11 @@ export class ProsemirrorAdapter {
     }
 
     private setupContentConverter() {
-        /* eslint-disable multiline-ternary */
-        this.contentConverter =
-            this.contentType === 'markdown'
-                ? new markdownConverter()
-                : new HTMLConverter();
-        /* eslint-enable multiline-ternary */
+        if (this.contentType === 'markdown') {
+            this.contentConverter = new markdownConverter();
+        } else if (this.contentType === 'html') {
+            this.contentConverter = new HTMLConverter();
```

Comment:
Nitpick: This isn't purely a refactor or code style change, since the existing behaviour would default to `HTMLConverter` if `this.contentType` was anything other than `'markdown'`, while this new behaviour is defaulting to `undefined` if `this.contentType` is neither `'markdown'` nor `'html'`.